apps/demo/src/pages/common/common-page.module.ts

```diff
@@ -0,0 +1,13 @@
+import {NgModule} from '@angular/core';
+import {CommonPageComponent} from './common-page.component';
+import {RouterModule} from '@angular/router';
+import {CommonModule} from '@angular/common';
+
+@NgModule({
+    declarations: [CommonPageComponent],
+    imports: [
```

Comment:
nit: I think imports should go first, this makes more sense semantically imo

Reply:
fixed